includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -465,6 +464,33 @@ protected function get_row_data( $item ) {
 					$user_average_grade = Sensei_Utils::quotient_as_absolute_rounded_number( $grade_total, $grade_count, 2 );
 				}
 
+				// Get the last lesson activity.
+				$last_activity_comment = Sensei_Utils::sensei_check_for_activity(
+					array(
+						'number'  => 1,
+						'user_id' => $item->ID,
+						'type'    => 'sensei_lesson_status',
+					),
+					true
+				);
+
+				$last_activity = null;
+				if ( $last_activity_comment ) {
+					if ( $this->csv_output ) {
+						$last_activity = $last_activity_comment->comment_date;
```

Comment:
Should we use GMT here or using WP timezone is good?

Reply:
The date registered was in GMT. Code updated in 510c9dddeae536273c6dfb322c1771ff68f5cb54.